stock_picking_auto_create_lot_biochousa/views/product_views.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2021 Solvos Consultoría Informática - Christian Santamaría
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <record id="view_template_property_form" model="ir.ui.view">
+        <field name="name">Product template exception eco prod</field>
+        <field name="model">product.template</field>
+        <field name="inherit_id" ref="stock.view_template_property_form" />
```

Comment:
The current code works, but the proper inheritance should be the following, since `auto_create_lot` field is added by `stock_picking_auto_create_lot.view_template_property_form` view:
```suggestion
        <field name="inherit_id" ref="stock_picking_auto_create_lot.view_template_property_form" />
```